tensorflow/g3doc/tutorials/deep_cnn/index.md

```diff
@@ -32,10 +32,14 @@ The CIFAR-10 tutorial demonstrates several important constructs for
 designing larger and more sophisticated models in TensorFlow:
 
 * Core mathematical components including [convolution](
-../../api_docs/python/nn.md#conv2d), [rectified linear activations](
-../../api_docs/python/nn.md#relu), [max pooling](
-../../api_docs/python/nn.md#max_pool) and [local response normalization](
-../../api_docs/python/nn.md#local_response_normalization).
+../../api_docs/python/nn.md#conv2d) ([wiki](
+https://en.wikipedia.org/wiki/Convolution)), [rectified linear activations](
+../../api_docs/python/nn.md#relu) ([wiki](
+https://en.wikipedia.org/wiki/Rectifier_(neural_networks))), [max pooling](
+../../api_docs/python/nn.md#max_pool) ([wiki](
+https://en.wikipedia.org/wiki/Convolutional_neural_network#Pooling_layer))
+and [local response normalization](
+../../api_docs/python/nn.md#local_response_normalization) ([quora](https://www.quora.com/What-is-local-response-normalization)).
```

Comment:
@vrv , yeah, that's a good idea
maybe directly link to AlexNet paper with chapter number, like 
(Chapter 3.3 in [AlexNet paper](http://www.cs.toronto.edu/~fritz/absps/imagenet.pdf))
This is more official than quora.